accesscontroltool-package/src/main/META-INF/vault/filter.xml

```diff
@@ -1,4 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
-	<filter root="/apps/netcentric/actool"/>       
+	<filter root="/apps/netcentric/actool"/>  
+	<filter root="/home/users/system/netcentric/actool/acl/ynUbs6nnylcpRBs0eXdv"/>

```

Comment:
Also this is still using the cryptic name. It should use the name "actool-service" or similar (in the same way as e.g. "authentication-service")